modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase

```diff
@@ -0,0 +1,160 @@
+				taxId = 12345,
+				zipCode = "12.345-67");
+		}
+
+		task ("When the employee user create a request") {
+			EVPRequest.fillRequestForm(
+				grantAmount = 20,
+				grantRequestType = "Proposed",
+				organization = "Organization Test",
+				phoneNumber = 12345,
+				requestBehalf = "Global/Local Poor",
+				requestDescription = "Request Description Test",
+				requestPurposes = "Basic Education",
+				requestType = "Grant");
+
+			EVPRequest.viewSuccessMessageRequest();
+		}
+
+		task ("Then the employee user view a Request Created and also view Organization Created") {
+			EVPNavigator.goToNavItem(navItem = "Request");
+
+			EVPUtils.viewStatus(
+				organizationName = "Organization Test",
+				status = "Awaiting Approval On EVP");
+
+			EVPNavigator.goToNavItem(navItem = "Organization");
+
+			EVPUtils.viewStatus(
+				organizationName = "Organization Test",
+				status = "Awaiting Approval On EVP");
+		}
+	}
+
+	@description = "LPS-162894 - Verify that after submit the form, when click on 'Done' button, will be redirected back to 'Organization' page"
+	@priority = 2
+	test CanSeeRedirectionToOrganizationPageAfterCreateOrganization {
+		task ("Given that the Employee user go to the organization page") {
+			EVPOrganization.goToCreateOrganization();
+		}
+
+		task ("When the employee user create an organization") {
+			EVPOrganization.fillOrganizationForm(
+				bankingInfo = "Bank Test",
+				city = "City Test",
+				contactEmail = "test@liferay.com",
+				contactName = "Contact Test",
+				contactPhone = 12345,
+				country = "Country Test",
+				done = "true",
+				organizationName = "Organization Test",
+				organizationSite = "www.test.com.br",
+				smallDescription = "Description Test",
+				state = "State Test",
+				street = "Street Test",
+				taxId = 12345,
+				zipCode = "12.345-67");
+		}
+
+		task ("Then The employee user is Redirection to page Organization") {
+			EVPNavigator.assertActiveBreadcrumb(activeItem = "Organization");
+		}
+	}
+
+	@description = "LPS-162894 - Verify that the user can create a new organization and a 'Thank you' message will be displayed"
+	@priority = 5
+	test CanseeSuccessMessageAfterCreateOrganization {
+		task ("Given that the Employee user go to organization page") {
+			EVPOrganization.goToCreateOrganization();
+		}
+
+		task ("When the Employee user create an organization") {
+			EVPOrganization.fillOrganizationForm(
+				bankingInfo = "Bank Test",
+				city = "City Test",
+				contactEmail = "test@liferay.com",
+				contactName = "Contact Test",
+				contactPhone = 12345,
+				country = "Country Test",
+				done = "true",
+				organizationName = "Organization Test 1",
+				organizationSite = "www.test.com.br",
+				smallDescription = "Description Test",
+				state = "State Test",
+				street = "Street Test",
+				taxId = 12345,
+				zipCode = "12.345-67");
+		}
+
+		task ("Then can see the message of successfully is displayed") {
+			EVPOrganization.viewSuccessMessageOrganization();
+		}
```

Comment:
Please change this line: `VPOrganization.viewSuccessMessageOrganization();` to `EVPUtils.viewSuccessMessage();`